.conda/condarc.yaml

```diff
@@ -2,6 +2,4 @@ channels:
   - sleap-deps/label/dev
   - conda-forge
   - nvidia
-  - https://conda.anaconda.org/sleap/label/dev
-  - sleap
```

Comment:
Why are we removing these?

Reply:
we are no longer using them, or am I missing something: https://github.com/talmolab/sleap/blob/3560663ebd24902b85e9f4321f2b4d50a97a9a23/.conda/meta.yaml#L114